plugin/core/windows.py

```diff
@@ -339,10 +339,10 @@ def start_async(self, config: ClientConfig, initiating_view: sublime.View) -> No
             self._new_session = session
         except Exception as e:
             message = "".join((
-                "Failed to start subprocess for {0}. Reason:\n\n",
-                "{1}\n\n",
-                "{0} will be disabled for this project. Enable {0} again by running ",
-                "\"LSP: Enable Language Server In Project\" from the Command Palette."
+                "Failed to start subprocess for {0}. {0} will be disabled for this window. ",
+                "Enable {0} again by running \"LSP: Enable Language Server In Project\" from the Command Palette.",
```

Comment:
- I think we can remove the "subprocess" word which is kinda a lower-level detail.
- The server name is repeated 3 times, once per-sentence.
Suggestion:
```suggestion
                "Failed to start \"{0}\" - disabling for this window. ",
                "Re-enable by running \"LSP: Enable Language Server In Project\" from the Command Palette.",
```